DEHPMatlab/DstController/DstController.cs

```diff
@@ -80,13 +90,24 @@ public class DstController : ReactiveObject, IDstController
         /// </summary>
         /// <param name="matlabConnector">The <see cref="IMatlabConnector"/></param>
         /// <param name="matlabParser">The <see cref="IMatlabParser"/></param>
-        public DstController(IMatlabConnector matlabConnector, IMatlabParser matlabParser)
+        /// <param name="statusBar">The <see cref="IStatusBarControlViewModel"/></param>
+        public DstController(IMatlabConnector matlabConnector, IMatlabParser matlabParser, IStatusBarControlViewModel statusBar)
         {
             this.matlabConnector = matlabConnector;
             this.matlabParser = matlabParser;
+            this.statusBar = statusBar;
             this.InitializeObservables();
         }
 
+        /// <summary>
+        /// Destructor of the <see cref="DstController"/>.
+        /// Handles the destruction of the loaded script.
+        /// </summary>
+        ~DstController()
```

Comment:
We never implement destructors. Is there no other way to implement the unloading?